src/_posts/platform/deployment/buildpacks/2000-01-01-jemalloc.md

```diff
@@ -59,23 +59,17 @@ Set this to select or pin to a specific version of jemalloc. The default is to
 use the latest stable version if this is not set. You will receive an error
 mentioning tar if the version does not exist.
 
-Default: **5.1.0**
+Default: **5.3.0**
 
 note: This setting is only used during app deployment. Changing it will
 require a code change to be deployed to take affect.
 
 ```
-scalingo env-set JEMALLOC_VERSION=3.6.0
+scalingo env-set JEMALLOC_VERSION=4.5.0
 ```
 
-Available Versions:
+Available versions:
 
 * 3.6.0
-* 4.0.4
```

Comment:
I also removed some versions to only keep the latest of each major version. I can revert if you don't like.

Reply:
praise: it's better like this :+1: